jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/WebSocketSessionState.java

```diff
@@ -129,7 +128,7 @@ public boolean onClosed(CloseStatus closeStatus)
      * </p>
      * <p>
      * This should only be called if there is an error directly before the call to
-     * {@link WebSocketCoreSession#closeConnection(CloseStatus, Callback)}.
+     * {@code WebSocketCoreSession.closeConnection(CloseStatus, Callback)}.
```

Comment:
the `.closeConnection(CloseStatus, Callback)` is not accessible from here and cannot be a javadoc link.